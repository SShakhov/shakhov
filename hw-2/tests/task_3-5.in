-1 0 -1 0.000004242 -123.123
