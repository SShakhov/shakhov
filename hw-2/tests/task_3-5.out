-125.122995758
